client/src/App.css

```diff
@@ -1,4 +1,7 @@
-
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
I understand why you need it but can you make sure it does not cause any issues in other pages

Reply:
@andrejskalnins have you resolved this conversation?